jupyter_ai_personas/pr_review_persona/persona.py

```diff
@@ -0,0 +1,232 @@
+import os
+from jupyter_ai.personas.base_persona import BasePersona, PersonaDefaults
+from jupyterlab_chat.models import Message
+from jupyter_ai.history import YChatHistory
+from agno.agent import Agent
+from agno.models.aws import AwsBedrock
+import boto3
+from agno.tools.github import GithubTools
+from agno.tools.reasoning import ReasoningTools
+from langchain_core.messages import HumanMessage
+from agno.tools.python import PythonTools
+from agno.team.team import Team
+from .ci_tools import CITools
+from .template import PRPersonaVariables, PR_PROMPT_TEMPLATE
+
+session = boto3.Session()
+
+class PRReviewPersona(BasePersona):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    @property
+    def defaults(self):
+        return PersonaDefaults(
+            name="PRReviewPersona",
```

Comment:
nit: @PRReviewPersona seems long to type, should we shorten this to `PRReviewer`?